plugins/reolink/src/main.ts

```diff
@@ -204,6 +204,8 @@ class ReolinkCamera extends RtspSmartCamera implements Camera, Reboot, Intercom,
                     if (!classes.length)
                         return;
 
+                    this.storageSettings.values.hasObjectDetector = ai;
```

Comment:
@koush regarding the empty response or if support = 0.. the code prior to this line will only fill up `classes` if support is 1.